vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/addon/spreadsheet/client/PopupButtonWidget.java

```diff
@@ -260,6 +273,7 @@ public boolean isPopupOpen() {
 
     public void closePopup() {
```

Comment:
Close Button is not visible, it should be as it's is in the v8 demo
![Screenshot 2022-06-13 at 12 56 13](https://user-images.githubusercontent.com/161853/173328953-d946f6b7-8129-4184-b458-a6f69ac053d6.png)

Reply:
Some styles were missing. Should be visible now.